pkg/beacon/beacon.go

```diff
@@ -74,12 +74,33 @@ func Initialize(
 	}
 
 	relayChain.OnRelayEntryRequested(func(request *event.Request) {
-		previousEntry := hex.EncodeToString(request.PreviousEntry[:])
+		currentEntryStartBlock, err := relayChain.CurrentEntryStartBlock()
+		if err != nil {
+			logger.Warningf("could not check current entry start block")
```

Comment:
`*current request start block` please. We need to rename this function. It's my fault, sorry.